static/js/import_wizard/types.ts

```diff
@@ -53,10 +53,12 @@ export interface MappingVal {
   // Column that holds the mapping values. Should be set if type is
   // MappingType.COLUMN
   column?: Column;
-  // When MappedThing is PLACE, the value corresponds to place property in KG.
-  placeProperty?: DCProperty;
-  // When MappedThing is PLACE, the value corresponds to place type in KG.
-  placeType?: DCType;
+  // Record of column idx to the place property (in KG) associated with that
+  // column or column header. Should be set if MappedThing is PLACE.
+  placeProperty?: Record<number, DCProperty>;
```

Comment:
I was previously puzzled with Record, and ran into issues when using it for "Mapping", since not all keys (MappedThing) may appear in an instance.

Reading https://betterprogramming.pub/typescripts-record-type-explained-691372b1a449, would the Index Signature type be more appropriate here?

Given we're not constraining the "number" values, and it allows a human-readable name for `number`, like `{ [columnIdx: number]: DCProperty }`.

Reply:
To confirm, there will be multiple entries here only if MappingType is COLUMN_HEADERS?

Reply:
that makes sense to me, done.

Correct, there should only be multiple entries if MappingType is COLUMN_HEADERS